node/pkg/watchers/near/watcher.go

```diff
@@ -200,14 +203,17 @@ func (e *Watcher) runObsvReqProcessor(ctx context.Context) error {
 			// Guardians currently run nodes for all shards and the API seems to be returning the correct results independent of the set senderAccountId but this could change in the future.
 			// Fixing this would require adding the transaction sender account ID to the observation request.
 			job := newTransactionProcessingJob(txHash, e.wormholeAccount)
-			e.schedule(ctx, job, time.Nanosecond)
+			err := e.schedule(ctx, job, time.Nanosecond)
+			if err != nil {
+				logger.Warn("near.schedule failed", zap.String("error", err.Error()))
+			}
 		}
 	}
 }
 
 func (e *Watcher) runTxProcessor(ctx context.Context) error {
 	logger := supervisor.Logger(ctx)
-	supervisor.Signal(ctx, supervisor.SignalHealthy)
```

Comment:
Do you add this back anywhere?

Reply:
Nope.. Not needed anymore since we are not creating different supervisor contexts